out1.txt

```diff
@@ -0,0 +1,13 @@
+ Index Info  Parent  Right sibling
```

Comment:
Just a suggestion, but you could do some pretty printing here, meaning that the columns could be centered under the headers